### DISCUSSION TITLE
Direct paths with davs

### DISCUSSION THREAD

**Discussion by tuffno** [croneter/PlexKodiConnect#1577] [Q&A]
Hello and thanks for this awsome plugin.

Im using Kodi on a few different locations over the internet using davs.
But syncing kodi library the traditional way over davs is extremely slow and not really usable. PKC did it in a minute which is really cool.
I would still love to play the contect with Direct Paths (with the DAVS:// paths so I dont involve the PMS during playback.
I add my sources using davs://user:pass@domain/movies

My question is will Direct Path work with other protocols than NFS and SMB like HTTPS or DAVS?
Or is Direct Play via plex my only option?

Thanks!

**Comment by piejanssens** [comment]
Direct Paths allows PKC to use the direct media file paths for playback. 
Any protocol supported by Kodi should work. 
If you use Kodi+PKC in different locations, then it's better not to use Direct Paths (as Kodi doesn't take care of transcoding for you and PMS via PKC can).
